services/QuillLMS/client/app/bundles/PremiumHub/containers/PremiumFilterableReportsContainer.tsx

```diff
@@ -290,8 +289,11 @@ export const PremiumFilterableReportsContainer = ({ accessType, adminInfo, locat
       <Filters
         {...filterProps}
       />
-      {shouldRenderDataExportContainer && <DataExportContainer {...sharedProps} />}
-      {shouldRenderUsageSnapshotsContainer && <UsageSnapshotsContainer {...sharedProps} />}
+      <Switch>
```

Comment:
I'm excited that we're able to use react-router here now! Any idea what changed? Also, it would be good to use [react-router v6 syntax](https://reactrouter.com/en/main/upgrading/v5) here if possible.

Reply:
I think it was an issue with using `component` rather than `render`